production/helm/loki/values.yaml

```diff
@@ -144,3 +144,7 @@ terminationGracePeriodSeconds: 30
 ## Tolerations for pod assignment
 ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
 tolerations: []
+
+# The values to set in the PodDisruptionBudget spec (minAvailable/maxUnavailable)
+# If not set then a PodDisruptionBudget will not be created
+podDisruptionBudget:
```

Comment:
Default values should be `{}`. Also, add a commented out example showing maxUnavailable and minUnavailable